packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js

```diff
@@ -58,11 +58,12 @@ class DemoWidget extends Component {
       composerActions,
       destinationId,
       initialActivity: isMeetOnly ? 'meet' : cookies.get('initialActivity') || 'message',
-      enableSpaceListFilter: cookies.get('enableSpaceListFilter') === 'true',
+      enableSpaceListFilter: cookies.get('enableSpaceListFilter') === 'true' || true,
```

Comment:
break what exactly? this is what we have in default in our README. the demo page should not contradict with what we have in README. it didn't break anything for me.

Reply:
also I remember asking you and you told me feel free to change them... and I did.

Reply:
If the user sets the config manually to false, when they return to the demo page, it will read from the cookie and see that it is 'false", but it will not actually set the state value to false because of the added `|| true`.

Reply:
I will update the code to not break the functionality.